quora-api/src/main/resources/application.yaml

```diff
@@ -10,13 +10,13 @@ spring:
 
   datasource:
     driverClassName: org.postgresql.Driver
-    url: jdbc:postgresql://localhost:5433/Quora
+    url: jdbc:postgresql://127.0.0.1:5432/mandeep
```

Comment:
don't check in this file. Since this is specific to your configuration.
It will create problem for others